packages/core-input/core-input.js

```diff
@@ -108,6 +110,7 @@ function setupExpand (input, open = input.getAttribute('aria-expanded') === 'tru
   requestAnimFrame(() => { // Fixes VoiceOver Safari focus jumping to parentElement
     input.nextElementSibling[open ? 'removeAttribute' : 'setAttribute']('hidden', '')
     input.setAttribute('aria-expanded', open)
+    if (open) onFilter(input, { relatedTarget: input.nextElementSibling })
```

Comment:
Hm, innser at dette fyrer av et ajax kall for hver focus på inputen. Kanskje finne en annen måte enn `onFilter`. Forslag; Hva med å flytte ` && index < limit` til `setupItem` funksjonen - altså skjule items da? Da er vi heeelt sikker på limiting, og det skjer både på init og filter :)